isis/src/qisis/objs/JigsawSetupDialog/JigsawSetupDialog.h

```diff
@@ -157,6 +157,7 @@ namespace Isis {
   private:
```

Comment:
Make sure to add a history entry to JigsawSetupDialog.h about adding/implementing the applySettingsPushButton_clicked()